processador-bd1pediat.py

```diff
@@ -1,77 +1,36 @@
-# -*- coding: utf-8 -*-
-"""Untitled0.ipynb
-
-Automatically generated by Colaboratory.
-
-Original file is located at
-    https://colab.research.google.com/drive/1km-HmqF_9R5UE8Aij73TksQwKHMvx_Te
-
-Aqui começo realizando o ```import``` de algumas bibliotecas úteis para a manipulação, processamento e apresentação dos dados desse script.
-"""
-
 import pandas as pd
 import numpy as np
 import scipy.stats
-
-"""Utilizo uma versão do dataset ```BD1PEDIAT.xls``` hospedado no meu Drive institucional, isso me permite acessar os dados sem necessariamente precisar ter o dataset salvo em memória. Isso também me permitiu utilizar o Google Colab com mais facilidade."""
-
-from google.colab import drive
-
-drive.mount('/content/drive/')
-
-"""Aqui faço o apontamento para o dataset ```BD1PEDIAT.xls``` e separo as informações por colunas."""
-
-data = pd.read_excel(r'/content/drive/MyDrive/Datasets (1)/BD1Pediat.xls', usecols = ['T_GEST', 'PESO', 'ESTATURA', 'PC', 'PT', 'SEXO', 'SANGUE', 'RH', 'ANOMALIA'])
-
-"""Aqui necessito instalar uma versão específica da biblitoeca ```xlrd``` para utilizar uma função da biblioteca ```scipy.stats```"""
-
 pip install xlrd==1.2.0
 
-"""Função ```mean_confidence_interval``` é responsável por retornar a média, o limite inferior e o limite superior."""
+data = pd.read_excel(r'./BD1Pediat.xls', usecols = ['T_GEST', 'PESO', 'ESTATURA', 'PC', 'PT', 'SEXO', 'SANGUE', 'RH', 'ANOMALIA'])
```

Comment:
```suggestion
usecols = ['T_GEST', 'PESO', 'ESTATURA', 'PC', 'PT', 'SEXO', 'SANGUE', 'RH', 'ANOMALIA']
data = pd.read_excel(r'./BD1Pediat.xls', usecols)
```

Fica melhor pra interpretar assim

Reply:
Concordo, vai ficar mais fácil pra mudar caso o dataset mude e seja necessário trocar os campos no futuro.